main.qml

```diff
@@ -1465,16 +1427,18 @@ ApplicationWindow {
         }
     }
 
-    // Confrirmation aka question dialog
-    StandardDialog {
+    // Transaction confirmation popup
+    TxConfirmationDialog {
+        // dynamically change onclose handler
+        id: txConfirmationPopup
         z: parent.z + 1
-        id: transactionConfirmationPopup
         onAccepted: {
             var handleAccepted = function() {
                 // Save transaction to file if view only wallet
                 if (viewOnly) {
                     saveTxDialog.open();
                 } else {
+                    open();
```

Comment:
Why do you want invoke `open` here?

Reply:
- User clicks in Confirm button
- txConfirmation popup is closed
- password dialog is displayed.
- txConfirmationPopup is opened while the transaction is not commited yet
- HandleTransactionConfirmed() displays "Sending transaction..." status. Normally it only flashes, but sometimes it can take some time to send the transaction, so we need to display this sending message:
```
            txConfirmationPopup.bottomText.text  = qsTr("Sending transaction...") + translationManager.emptyString;
```
- When transaction is commited (async), txConfirmationPopup is closed and SuccessfulTxDialog is opened (#3121)

Reply:
> txConfirmationPopup is opened while the transaction is not commited yet

That's not okay. A user already confirmed the tx.

Reply:
- We can leave as it is now (not perfect but working), or
- We can stop opening txConfirmationPopup to display the "Sending transaction..." message and later I send a PR adding "Sending transaction... message" to successfulTxPopup

Any other idea?

Reply:
The second option looks better to me.

Reply:
Ok, I updated main.qml and txConfirmationPopup is not opening for a second time. After this PR is merged I'll modify successfulTxPopup.qml to display "Sending message..." and error messages.